diff -r c5ee44e41355 mobile/android/exoplayer2/build.gradle
--- a/mobile/android/exoplayer2/build.gradle	Mon Apr 08 04:40:01 2024 +0000
+++ b/mobile/android/exoplayer2/build.gradle	Mon Apr 29 08:08:02 2024 -0400
@@ -34,10 +34,6 @@
         }
     }
 
-    publishing {
-        singleVariant("debug")
-    }
-
     namespace 'org.mozilla.geckoview.thirdparty'
 }
 
diff -r c5ee44e41355 mobile/android/geckoview/build.gradle
--- a/mobile/android/geckoview/build.gradle	Mon Apr 08 04:40:01 2024 +0000
+++ b/mobile/android/geckoview/build.gradle	Mon Apr 29 08:08:02 2024 -0400
@@ -155,10 +155,6 @@
         aidl = true
     }
 
-    publishing {
-        singleVariant('withGeckoBinariesDebug')
-    }
-
     namespace 'org.mozilla.geckoview'
 }
 
