diff -r 2df6d3fa14ca -r f0a2d70cf468 mfbt/SmallPointerArray.h
--- a/mfbt/SmallPointerArray.h	Mon Oct 28 08:39:49 2024 -0400
+++ b/mfbt/SmallPointerArray.h	Mon Oct 28 08:40:23 2024 -0400
@@ -201,9 +201,9 @@
  private:
   T** beginInternal() const {
     if (first()) {
-      static_assert(sizeof(T*) == sizeof(Element),
+      static_assert(sizeof(T*) == sizeof(SmallArrayElement),
                     "pointer ops on &first() must produce adjacent "
-                    "Element::mValue arms");
+                    "SmallArrayElement::mValue arms");
       return &first();
     }
 
@@ -259,7 +259,7 @@
   // When !mArray[0].mValue && mArray[1].mVector, mArray[1].mVector contains
   // the contents of an array of arbitrary size (even less than two if it ever
   // contained three elements and elements were removed).
-  union Element {
+  union SmallArrayElement {
     T* mValue;
     std::vector<T*>* mVector;
   } mArray[2];
