diff -r 0afa0a70f9ff -r 740e38722bb7 mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/fake/FakeHomepagePreview.kt
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/fake/FakeHomepagePreview.kt	Wed Oct 02 13:09:44 2024 -0400
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/fake/FakeHomepagePreview.kt	Wed Oct 02 13:11:35 2024 -0400
@@ -4,7 +4,6 @@
 
 package org.mozilla.fenix.home.fake
 
-import com.google.firebase.util.nextAlphanumericString
 import mozilla.components.browser.state.state.ContentState
 import mozilla.components.browser.state.state.TabSessionState
 import mozilla.components.concept.sync.DeviceType
@@ -124,7 +123,7 @@
                 add(
                     RecentTab.Tab(
                         TabSessionState(
-                            id = randomString(),
+                            id = "rAnDoM",
                             content = ContentState(
                                 url = URL,
                             ),
@@ -159,9 +158,4 @@
     private const val URL = "mozilla.com"
 
     private fun randomLong() = random.nextLong()
-
-    private fun randomString(length: Int = random.nextInt(from = 3, until = 11)) =
-        random.nextAlphanumericString(
-            length = length,
-        )
 }
