diff -r a34f5816040e -r 5cafe8ddd023 mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/fake/FakeHomepagePreview.kt
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/fake/FakeHomepagePreview.kt	Mon Oct 28 08:37:09 2024 -0400
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/fake/FakeHomepagePreview.kt	Mon Oct 28 08:38:19 2024 -0400
@@ -5,7 +5,6 @@
 package org.mozilla.fenix.home.fake
 
 import android.content.Context
-import com.google.firebase.util.nextAlphanumericString
 import mozilla.components.browser.state.state.ContentState
 import mozilla.components.browser.state.state.TabSessionState
 import mozilla.components.browser.state.state.recover.RecoverableTab
@@ -235,7 +234,7 @@
                 add(
                     RecentTab.Tab(
                         TabSessionState(
-                            id = randomString(),
+                            id = "rAnDoM",
                             content = ContentState(
                                 url = URL,
                             ),
@@ -329,9 +328,4 @@
     private const val URL = "mozilla.com"
 
     private fun randomLong() = random.nextLong()
-
-    private fun randomString(length: Int = random.nextInt(from = 3, until = 11)) =
-        random.nextAlphanumericString(
-            length = length,
-        )
 }
